OWNERS

```diff
@@ -4,3 +4,4 @@ approvers:
   - stlaz
   - sttts
   - deads2k
+components: apiserver-auth
```

Comment:
The example in the email says "component" rather than "components". Are you sure it is plural?